lionblon-hojun/src/compoents/Comment/CommentElement.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState, useEffect } from "react";
+
+const CommentElement = ({
+  comment,
+  handleCommentDelete,
+  handleEditComment,
+}) => {
+  /* TODO: 댓글을 수정하는 input의 value를 관리하기 위한 state 작성
+       Hint: 댓글의 내용을 저장하는 state와 수정 중인지 여부를 저장하는 state를 따로 만드는 게 좋겠죠? */
+  const [commentEditInput, setcommentEditInput] = useState(comment.content);
```

Comment:
```suggestion
  const [commentEditInput, setCommentEditInput] = useState(comment.content);
```
setState도 Camel Case로 작성해주세요!!